### PR TITLE
Bump to kind 0.11

### PR DESCRIPTION
**Pull request by skitt**
See https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.0 for
details. This bumps the supported K8s versions to 1.18.19, 1.19.11,
1.20.7, and adds support for 1.21 (1.21.1).

Depends on https://github.com/submariner-io/shipyard/issues/567
Depends on https://github.com/submariner-io/lighthouse/pull/548
Depends on https://github.com/submariner-io/submariner/pull/1371
Depends on https://github.com/submariner-io/submariner-charts/pull/145
Depends on https://github.com/submariner-io/submariner-operator/pull/1392
Depends on https://github.com/submariner-io/shipyard/issues/575

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Comment by skitt** [comment]
Hmm, the test matrix in dependent projects (_e.g._ the operator) means that this will break them...

File: .github/workflows/consuming.yml [hunk 1]
```diff
@@ -36,11 +36,11 @@ jobs:
...
           # Recentness of K8s versions are limited by kindest/node image releases
           - project: submariner-operator
-            k8s_version: 1.18.15
+            k8s_version: 1.18.19
           - project: submariner-operator
-            k8s_version: 1.19.7
+            k8s_version: 1.19.11
           - project: submariner-operator
-            k8s_version: 1.20.2
+            k8s_version: 1.20.7
```
Comment (mkolesnik) [inline .github/workflows/consuming.yml]: Shouldn't these be short versions? :)

Comment (skitt) [reply .github/workflows/consuming.yml]: They should now ;-). I wrote this patch before filing #567 as a result of the resulting downstream failures!

**Review by dfarrell07 — APPROVED** [review]
I think all the dependencies are merged, but needs a manual rebase.

**Review by mkolesnik — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**